2.x-centos7/Dockerfile

```diff
@@ -0,0 +1,281 @@
+FROM centos:7
+MAINTAINER mail@racktear.com
+
+RUN groupadd tarantool \
+    && adduser -g tarantool tarantool
+
+ENV TARANTOOL_VERSION=2.1.2-9-g72394afda \
+    TARANTOOL_DOWNLOAD_URL=https://github.com/tarantool/tarantool.git \
+    TARANTOOL_INSTALL_LUADIR=/usr/local/share/tarantool \
+    CURL_REPO=https://github.com/curl/curl.git \
+    CURL_TAG=curl-7_64_0 \
```

Comment:
Tarantool works correctly with libcurl 7.60 and above after 2.1.2-49-gc413011cc when threaded DNS resolver is enabled (it is default). Please, update the tarantool version (it is better option) or use curl-7.59.

Reply:
Updated to 2.2.0-240-gfa6bb2fe0

Reply:
Are you sure you want to try an unreleased version? I would suggest to put last 2.1 here. At the time of writing it is 2.1.2-52-g154345fc5.

Reply:
Ok. 2.1.2-52-g154345fc5